mvp-spec/README.md

```diff
@@ -1,111 +1,56 @@
-| [operator-api.md](operator-api.md)                                         | Operator API specification                                                                          |
-| [operator-design.md](operator-design.md)                                   | Design of the generation of Prebid SSO Data.                                                        |
-| [operator-design-alternative-swan.md](operator-design-alternative-swan.md) | Summary of the SWAN solution for generating PAF Data.                                               |
-| [operator-requirements.md](operator-requirements.md)                       | Requirements for the generation of the PAF Data.                                                    |
-| [operator-client.md](operator-client.md)                                   | Modules needed to connect to the operator                                                           |
-| [model/](model)                                                            | Data and messages model                                                                             |
-| [json-schemas/](json-schemas)                                              | Data and messages model in [JSON schema](https://json-schema.org/understanding-json-schema/) format |
-| `assets/` `model-updater/` `partials/` `partials-updater/`                 | Technical dependencies, please ignore                                                               |
-
-## Architecture
-
-PAF integrates in the existing digital marketing landscape and introduces a new actor: the "PAF operator".
+## Overview
 
 ```mermaid
 flowchart LR
     
     O(PAF Operator)
     click O href "#operator" "Operator API"
     
-    Ad(Ad server)
-    click Ad href "#ad-server" "Ad server"
-    
-    Advertiser("Advertiser website")
-    Advertiser --->|read user ids & preferences| O
-    click Advertiser href "#advertiser" "Advertiser"
+    Participant("Participant website")
     
-    Publisher("Publisher website")
-    
-    Publisher -->|read user ids & preferences| O
-    click Publisher href "#publisher" "Publisher"
-    
-    Publisher -.->|include| UI
-    Publisher -- start transaction --> SSP
-    Publisher -- get ad & audit logs --> Ad
+    Participant -->|read user ids & preferences| O
+    click Participant href "#publisher" "Participant"
     
+    Participant -- start transaction --> SSP
     SSP -- send transmission --> DSP
+    DSP -- send transmission response --> SSP
+    SSP -- send transmission response --> Participant    
+
     click SSP href "#ssp-supply-side-platform" "SSP"
     click DSP href "#dsp-demand-side-platform" "DSP"
-    
-    UI("User Interface Provider") -->|write user preferences| O
-    click UI href "#user-interface-provider" "UI provider"
 ```
 
-## Nodes
-
-### Advertiser website
```

Comment:
> I thought it was still useful to have a place where this "roles" are mentioned so that a particular actor (publisher, advertiser) could get "pointers" to relevant documents.

Agreed, I added on commit to put these "nodes" back. WDYT?